src/main/java/uk/gov/register/store/postgres/PostgresDriver.java

```diff
@@ -7,22 +7,21 @@
 import org.skife.jdbi.v2.Handle;
 import org.skife.jdbi.v2.tweak.HandleCallback;
 import org.skife.jdbi.v2.tweak.HandleConsumer;
-import uk.gov.register.core.Entry;
-import uk.gov.register.core.Item;
-import uk.gov.register.core.Record;
+import uk.gov.register.core.*;
 import uk.gov.register.db.*;
+import uk.gov.register.exceptions.NoSuchFieldException;
 import uk.gov.register.store.BackingStoreDriver;
 import uk.gov.verifiablelog.VerifiableLog;
 import uk.gov.verifiablelog.store.memoization.MemoizationStore;
 
 import java.time.Instant;
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 
 public abstract class PostgresDriver implements BackingStoreDriver {
 
     protected final MemoizationStore memoizationStore;
+    private final HashMap<String, ArrayList<String>> registerFieldLookup;
+
```

Comment:
This is no longer used. There should be no changes to this file as part of this PR.